app/src/main/java/emu/skyline/input/onscreen/OnScreenEditActivity.kt

```diff
@@ -80,6 +80,8 @@ class OnScreenEditActivity : AppCompatActivity() {
             Pair(R.drawable.ic_edit, editAction),
             Pair(R.drawable.ic_zoom_out, { binding.onScreenControllerView.decreaseScale() }),
             Pair(R.drawable.ic_zoom_in, { binding.onScreenControllerView.increaseScale() }),
+            Pair(R.drawable.ic_opacity_minus) { binding.onScreenControllerView.changeOpacity(false) },
+            Pair(R.drawable.ic_opacity_plus) { binding.onScreenControllerView.changeOpacity(true) },
```

Comment:
```suggestion
            Pair(R.drawable.ic_opacity_minus) { binding.onScreenControllerView.changeOpacity(OPACITY_DELTA_STEP) },
            Pair(R.drawable.ic_opacity_plus) { binding.onScreenControllerView.changeOpacity(-OPACITY_DELTA_STEP) },
```

Create a `OPACITY_DELTA_STEP` constant at the top of this class with value 10.